server/lib/routes/get-index.js

```diff
@@ -11,9 +25,20 @@ module.exports = function (config) {
   route.path = '/';
 
   route.process = function (req, res) {
+    var time = Date.now();
+    var flowId = crypto.randomBytes(16).toString('hex');
+
     res.render('index', {
+      flowBeginTime: time,
+      flowId: flowId,
       staticResourceUrl: STATIC_RESOURCE_URL
```

Comment:
Hrrm, no idea. Hopefully @shane-tomlinson can clarify for us when he's in tomorrow.